src/test/scala/com/databricks/spark/sql/perf/mllib/MLLibSuite.scala

```diff
@@ -1,10 +1,29 @@
 package com.databricks.spark.sql.perf.mllib
 
-import org.scalatest.FunSuite
+import org.scalatest.{BeforeAndAfterAll, FunSuite}
 
+import org.apache.spark.{SparkConf, SparkContext}
 import org.apache.spark.sql.Row
 
-class MLLibTest extends FunSuite {
+class MLLibSuite extends FunSuite with BeforeAndAfterAll {
+
+  private var sc: SparkContext = _
+
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+    val sparkConf = new SparkConf().setAppName("MLlib QA").setMaster("local[2]")
+    sc = SparkContext.getOrCreate(sparkConf)
```

Comment:
This is fine with me since it works, but you could use a SparkSession since we don't need to support Spark 2.0 anymore.